spec/request/posts_request_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'Get /index' do
+    it 'returns http success' do
+      get '/users/:user_id/posts'
+      expect(response).to have_http_status(:success)
+      expect(response).to have_http_status(200)
+    end
+
+    it 'returns the expected content' do
+      get '/users/:user_id/posts'
+      expect(response.body).to include('This is Post index')
+    end
```

Comment:
- [ ] Great job working on the tests but I think it would be great if you add all the tests required and to make sure if all the actions are working including this `If a correct template was rendered.` I highly recommend adding it in index and show page for posts as well💯🔥🚀✅

![image](https://github.com/CesarHerr/blogApp/assets/58553711/2dcebee5-f0cf-48a0-ba4a-e68436c7ca16)